src/commands/env/create/sandbox.ts

```diff
@@ -131,6 +132,11 @@ export default class CreateSandbox extends SandboxCommandBase<SandboxProcessObje
     'no-prompt': Flags.boolean({
       summary: messages.getMessage('flags.noPrompt.summary'),
     }),
+    'no-track-source': Flags.boolean({
```

Comment:
This still feels like arm twisting to me. Any way to flip this to the positive? `--track-source` with allowNo: true?

Reply:
that worked fine for scratch where the default is "track stuff".

You **could** do the same thing here, but then you'd require users to `--no-track-source` for sandboxes that don't support tracking (Partial/Full) and have to throw if they didn't include it.

Or you'd be allowing them to use `--track-source` and then ignoring it for sandboxes that can't.

As is, I think this is the cleanest way to preserve the intent of this feature, which is "opt out of the default"

And it's consistent with other env create `--no-namespace` and `--no-ancestors` that are opt-out-ish